content/laptop-battery-thresholds.md

```diff
@@ -16,11 +16,13 @@ tableOfContents: true
 
 Charging thresholds allow your System76 laptop to avoid charging the battery until it has dropped below a lower bound (the start threshold), and to stop charging when it reaches an upper bound (the end threshold). This is useful when your laptop is plugged into an AC power adapter for extended periods of time, as it prevents unnecessary micro-charging that would reduce battery longevity.
 
-To determine if your laptop has Open Firmware or proprietary firmware, see [this article](/articles/open-firmware-systems). (If a system has Open Firmware, then it must also have Open EC to work with charging thresholds.) See [Charging Thresholds](#configuring-charging-thresholds-open-firmware) for Open Firmware systems or [FlexiCharger](#configuring-flexicharger-proprietary-firmware) for proprietary firmware systems.
+The default charging profile for System76 laptops with Open Firmware have a start threshold of 90 and a stop threshold of 100. This increases battery longevity, while keeping the battery very close to a full charge.
```

Comment:
"profile" is singular, so it should be "has" rather than "have."

I don't think a comma before "while" is necessary in this case, but I don't have a problem with it stylistically.

Reply:
Corrected. I just removed the comma.